user_manual/docs/firmware.md

```diff
@@ -5,39 +5,47 @@ Updating Firmware
 
 # Overview {#update-overview}
 
-The **Firmware** is the code that runs on the board.
+The **Firmware** is the OLE code that runs on the device during normal operating
+conditions.
 
-The **Bootloader** is a mode that allows the firmware to be written to the board
+The **Bootloader** is a mode that allows the firmware to be written to the
```

Comment:
That's probably over the head of most users - they don't think of Bootloaders as 'code' .

I've changed it a bit to try to make it clear.